dataprep/clean/clean_duplication_utils.py

```diff
@@ -0,0 +1,323 @@
+"""
+Common functions and classes for the clean_duplication function.
+"""
+
+import string
+import unicodedata
+import collections
```

Comment:
Do you think it's better to only import the functions used in this module? E.g. `from collections import defaultdict`.

Reply:
Ya